exercises/practice/minesweeper/.docs/instructions.md

```diff
@@ -0,0 +1,32 @@
+# Instructions
+
+Add the mine counts to a completed Minesweeper board.
+
+Minesweeper is a popular game where the user has to find the mines using numeric hints that indicate how many mines are directly adjacent (horizontally, vertically, diagonally) to a square.
+
+In this exercise you have to create some code that counts the number of mines adjacent to a given empty square and replaces that square with the count.
```

Comment:
```suggestion
In this exercise you have to create some code that counts the number of mines adjacent to a given empty square and replace that square with the count.
```

I realise this probably comes from the problem-specifications, but this looks like a typo to me.

Reply:
it's a little awkward, but I think it's correct:

> you have to create some code the counts [...] and replaces [...]